tests/compute/test_serialize.py

```diff
@@ -174,13 +177,103 @@ def test_serialize_empty_dict():
 
     load_tensor_dict = load_tensors(path)
     assert isinstance(load_tensor_dict, dict)
-    assert len(load_tensor_dict) == 0   
+    assert len(load_tensor_dict) == 0
+
+    os.unlink(path)
+
+
+def test_load_old_files1():
+    loadg_list, _ = load_graphs(os.path.join(
+        os.path.dirname(__file__), "data/1.bin"))
+    idx, num_nodes, edge0, edge1, edata_e1, edata_e2, ndata_n1 = np.load(
+        os.path.join(os.path.dirname(__file__), "data/1.npy"), allow_pickle=True)
+
+    load_g = loadg_list[idx]
+    load_edges = load_g.all_edges('uv', 'eid')
+
+    assert np.allclose(F.asnumpy(load_edges[0]), edge0)
+    assert np.allclose(F.asnumpy(load_edges[1]), edge1)
+    assert np.allclose(F.asnumpy(load_g.edata['e1']), edata_e1)
+    assert np.allclose(F.asnumpy(load_g.edata['e2']), edata_e2)
+    assert np.allclose(F.asnumpy(load_g.ndata['n1']), ndata_n1)
+
+
+def test_load_old_files2():
+    loadg_list, labels0 = load_graphs(os.path.join(
+        os.path.dirname(__file__), "data/2.bin"))
+    labels1 = load_labels(os.path.join(
+        os.path.dirname(__file__), "data/2.bin"))
+    idx, edges0, edges1, np_labels = np.load(os.path.join(
+        os.path.dirname(__file__), "data/2.npy"), allow_pickle=True)
+    assert np.allclose(F.asnumpy(labels0['label']), np_labels)
+    assert np.allclose(F.asnumpy(labels1['label']), np_labels)
+
+    load_g = loadg_list[idx]
+    load_edges = load_g.all_edges('uv', 'eid')
+    assert np.allclose(F.asnumpy(load_edges[0]), edges0)
+    assert np.allclose(F.asnumpy(load_edges[1]), edges1)
+
+
+def create_heterographs(index_dtype):
+    g_x = dgl.graph(([0, 1, 2], [1, 2, 3]), 'user',
+                    'follows', index_dtype=index_dtype, restrict_format='any')
+    g_y = dgl.graph(([0, 2], [2, 3]), 'user', 'knows', index_dtype=index_dtype, restrict_format='csr')
+    g_x.nodes['user'].data['h'] = F.randn((4, 3))
```

Comment:
What will happen if there is an empty data e.g., data['h'] = None?

Reply:
It's not viable to assign `None` to ndata of heterograph. It must be a backend tensor